src/components/background/background.scss

```diff
@@ -57,3 +57,11 @@
 [class*="bg--black"] {
   border-color: rgba(255,255,255,.325);
 }
+
+// override gray link color.
+[class*="bg--white"] :is(p, ul, ol, dl, cite) a,
+[class*="bg-"] [class*="bg--white"] :is(p, ul, ol, dl, cite) a,
+[class*="bg--gray"] :is(p, ul, ol, dl, cite) a,
+[class*="bg-"] [class*="bg--gray"] :is(p, ul, ol, dl, cite) a {
```

Comment:
I'm wondering about the quotes here. I notice below that there are no quotes. Don't have a strong feeling either way, but want to be consistent.

Reply:
Good point, maybe we should use quotes?  MDN docs use them https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors.

Reply:
Sounds good to me.